transforms/scoped-jsx.js

```diff
@@ -73,4 +73,26 @@ const deprecatedReactChildTransform = (file, api) => {
 	return file.source;
 };
 
-module.exports = deprecatedReactChildTransform;
+/**
+ * @param {import('jscodeshift').API['jscodeshift']} j
+ * @param {import('jscodeshift').Collection} ast
+ */
+function jsxImportedFromJsxRuntime(j, ast) {
+	return (
+		ast
+			.find(j.ImportSpecifier, {
+				imported: { name: "JSX" },
+				local: { name: "JSX" },
+			})
+			.filter((path) => {
+				return (
+					path.parent &&
+					path.parent.node.type === "ImportDeclaration" &&
+					(path.parent.node.source.value === "react/jsx-runtime" ||
+						path.parent.node.source.value === "react/jsx-dev-runtime")
```

Comment:
I think we don't need to check for import sources at all. If you imported `JSX` already, that was the `JSX` you've been using all along so we shouldn't mess with that.

Reply:
Good point I took care of that in 355375d4d9c589e200b182d3c98e0238de51cca7